src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task03.java

```diff
@@ -37,6 +37,21 @@ public static void main(String[] args) {
     static String getColorHealthPoint(int currentHealthPoint, int maxHealthPoint) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        String hpColour;
+        final int twentyFivePercent = 25;
+        final int fiftyPercent = 50;
```

Comment:
"Violation: 'NAME_CONSTANT' was not used for constant naming
Solution: Rename constant"